lib/commands/screenshots.js

```diff
@@ -17,7 +17,8 @@ commands.getScreenshot = async function getScreenshot () {
   };
 
   // if we've specified an mjpeg server, use that
-  if (this.mjpegStrem) {
+  if (this.mjpegStream) {
```

Comment:
this is a bug that would not have been possible in some other languages :-/ i'm guessing it's my typo!